docker/createml.go

```diff
@@ -200,15 +200,54 @@ func PutManifestList(a *types.AuthInfo, yamlInput types.YAMLInput, ignoreMissing
 	}
 	defer resp.Body.Close()
 
+	var finalDigest string
 	if statusSuccess(resp.StatusCode) {
 		dgstHeader := resp.Header.Get("Docker-Content-Digest")
 		dgst, err := digest.Parse(dgstHeader)
 		if err != nil {
 			return "", err
 		}
-		return string(dgst), nil
+		finalDigest = string(dgst)
+	} else {
+		return "", fmt.Errorf("Registry push unsuccessful: response %d: %s", resp.StatusCode, resp.Status)
+	}
+	// if the YAML includes additional tags, push the added tag references. No other work
+	// should be required as we have already
```

Comment:
Was this comment intended to sound like a cliffhanger?  (feels like it ends in the middle of a sentence)